src/components/DiaryCard.js

```diff
@@ -39,7 +39,7 @@ const DiaryCard = ({tittle, subtitle, description , color}) => {
             </Typography>
           </CardContent>
           <CardActions>
-          <Button size="small" onClick={isShowMore}>{isShowMore ? 'SHOW MORE' : 'SHOW LESS'}</Button>
+          <Button size="small" onClick={toggleShowMore}>{toggleShowMore ? 'SHOW MORE' : 'SHOW LESS'}</Button>
```

Comment:
There is a mistake here. `toggleShowMore` is a function. You have used it as a boolean. It is wrong. Fix that.